packages/mint-components/src/components/tax-and-cash/sqm-tax-form.feature

```diff
@@ -0,0 +1,226 @@
+      | brandCountry | country | participantType       | autoSelectedForm |
+      | US           | CA      | individualParticipant | W8-BEN           |
+      | US           | CA      | company               | W8-BEN-E         |
+      | US           | MX      | individualParticipant | W8-BEN           |
+      | US           | MX      | company               | W8-BEN-E         |
+
+  @minutia
+  Scenario Outline: Participants based in the US working with non-US brands have to fillout the W9 docusign form
+    Given a brand based in <brandCountry>
+    And the user selects <country>
+    When they view step 3
+    Then the <autoSelectedForm> is displayed
+
+    Examples:
+      | brandCountry | country | autoSelectedForm |
+      | MX           | US      | W9               |
+      | UK           | US      | W9               |
+      | AUS          | US      | W9               |
+
+  @minutia
+  Scenario: Participant finishes tax form flow and skipped Docusign form
+    Given they are on step 2
+    And they were not required to fillout a tax form
+    And they press Continue
+    Then they proceed to step 4
+    And complete their payout details
+    And they press Continue
+    Then they see the Dashboard page with no Tax Document present
+
+  @minutia
+  Scenario: Participant finishes tax form flow and sees status of their tax form submission
+    Given they are on step 3
+    And finishes filling out the Docusign form
+    And they press Continue
+    Then they proceed to step 4
+    And complete their payout details
+    And they press Continue
+    Then hey see the Dashboard page
+
+  @minutia
+  Scenario: A general error banner appears upon form submission request failing
+    When the participant completes a form with their information
+    And they press "Continue" to submit the form
+    Then a request is made to save the form data
+    But the request fails
+    Then a general error banner appears with <generalTitle> and <generalDescription>
+
+    Examples:
+      | generalTitle                                    | generalDescription                                                                       |
+      | There was a problem submitting your information | Please review your information and try again. If this problem continues, contact Support |
+
+  @minutia
+  Scenario: A loading error banner appears when a form fails to load
+    When the participant views a form step
+    Then a request is made to fetch the form data
+    But the request fails
+    Then a loading error banner appears with <loadingErrorAlertHeader> and <loadingErrorAlertDescription>
+
+    Examples:
+      | loadingErrorAlertHeader               | loadingErrorAlertDescription                                                       |
+      | There was a problem loading your form | Please refresh the page and try again. If this problem continues, contact Support. |
+
+  @minutia
+  Scenario: A general error banner appears upon form load request failing
```

Comment:
This needs to be a scenario outline or have the examples brought into the spec. Doesn't really need to be a scenario outline if its just one case